.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Test workflow
+
+on:
+  push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: g++ -Wall main.cpp Unit.cpp Game.cpp
+      - run: chmod +x test.sh
```

Comment:
ezt ha egyszer elintezed lokalisan, vbecommitolod, akkor letudva szerintem.

Reply:
Kipróbáltam még egyszer, de valamiért ha lokálisan megcsinálom és commitolom nem elég neki. Nem igazán értem miért. Ezért raktam be ide, hogy futtasa akkor.

Reply:
Jol sejtem, hogy windows, ugye?

Reply:
Hát, igen. Windows-on futtatok wsl-t, abban próbáltam.

Reply:
Hmm.. koszi az infot. Lattam errol neten diszkussziot, de most nem olvasok utana. Fix mondjuk tuti van. Majd azert lesd meg, lesz meg par script, amit futtatni kell.